applications/FluidDynamicsApplication/tests/cpp_tests/test_weighted_divergence_calculation.cpp

```diff
@@ -0,0 +1,112 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Tosi
+//
+//
+
+// System includes
+#include <set>
+
+// External includes
+
+// Project includes
+#include "testing/testing.h"
+#include "containers/model.h"
+#include "includes/model_part.h"
+#include "includes/cfd_variables.h"
+
+// Application includes
+#include "custom_elements/navier_stokes.h"
+#include "custom_elements/fluid_element.h"
```

Comment:
Why do you need the `fluid_element.h`? The `navier_stokes.h` already includes it.

Reply:
Done.